sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -629,22 +629,56 @@ def estimate_size(self, unused_value, nested=False):
 
 class TimerCoderImpl(StreamCoderImpl):
   """For internal use only; no backwards-compatibility guarantees."""
-  def __init__(self, payload_coder_impl):
+  def __init__(self, key_coder_impl, window_coder_impl, tag_coder_impl):
```

Comment:
why do you have to specify the tag_coder_impl, shouldn't it always be Python's string utf8 coder?

Reply:
Yes it should be as long as we agree on that the tag should be `string`.